aospy_synthetic/run.py

```diff
@@ -50,6 +52,9 @@ def __init__(self, name='', description='', proj=False,
         self.data_in_direc = self._set_direc(data_in_direc, ens_mem_prefix,
                                              ens_mem_ext, ens_mem_suffix)
 
+    def __hash__(self):
+        return hash((str(type(self)), self.name, self._parent))
+
     def __str__(self):
         return 'Run instance "%s"' % self.name
 
```

Comment:
I had completely forgotten about that code -- thanks for pointing it out.  Yes, for the purposes of this PR I think it is safe to delete it.  I may save that fragment in another location offline to jog my memory when we work on some front-end things (that's what that code was aimed towards), but it's pretty crude.